app/controllers/site_controller.rb

```diff
@@ -46,33 +46,61 @@ def index
 
   def load_habitat
     @habitat = Habitat.where(name: params['habitat'] || 'warmwater').first
-    @habitat ||= Habitat.where(name: 'coralreef').first
+    @habitat ||= Habitat.where(name: 'coralreefs').first
     @habitat_type = @habitat.type
   end
 
   def load_charts_data
-    top_five_countries = StaticStat.where(habitat_id: @habitat.id)
-                                   .order('total_value DESC')
-                                   .first(5)
-    arbitrary_value = top_five_countries.first.total_value.to_f * 1.05
+
+    query = <<-SQL
+      SELECT DISTINCT geo_entities.id AS geo_entities_id,
+                      geo_entities.iso3 AS iso3,
+                      geo_entities.name AS name,
+                      geo_entity_stats.habitat_id as habitat_id,
+                      geo_entity_stats.total_value AS total_value
+
+                      FROM geo_entities
+                      INNER JOIN geo_entity_stats ON geo_entities.id = geo_entity_stats.geo_entity_id
+                      INNER JOIN habitats ON habitats.id = geo_entity_stats.habitat_id
+                      WHERE geo_entities.iso3 IS NOT NULL
+                      AND habitat_id = #{@habitat.id}
+                      ORDER BY total_value DESC
+
```

Comment:
I know this is not finished, but just a note for the future.
You can use `LIMIT 5` after the `ORDER BY` to fetch the first 5 elements only instead of doing that with ActiveRecord